### PR TITLE
AWS Bdoc draft

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**
Draft will be created here.
While connector is not GA, documentation will be copied from here into the Marketplace.
This PR will be published when the connector is made GA

### CODE REVIEW

**Comment by katarzyna-koltun-mx** [comment]
This is a draft template for the team to fill out. It should not be published until the service is GA and platform-supported.

File: content/en/docs/appstore/connectors/aws/amazon-bedrock.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+3. In the **App Explorer**, double-click the **Settings** for your app.
+4. In the **App Settings** dialog, in the **Configurations** tab, edit or create an authentication profile.
+    If you have multiple sets of AWS credentials, or if you want to use both static and session credentials for different use cases, create separate authentication profiles for each set of credentials.
+5. In the **Edit Configuration** dialog, in the **Constants** tab, click **New** to add the constants required for the configuration.
+6. In the **Select Constants** dialog, find and expand the **AmazonBedrockConnector** > **ConnectionDetails** section.
+7. Depending on your selected authentication type, configure the required parameters for the **StaticCredentials** or **SessionCredentials**.
+
+    | Credentials type | Parameter | Value |
+    | --- | --- | --- |
+    | Any | **UseStaticCredentials** | **true** if you want to use static credentials, or **false** for session credentials |
```
Comment (trongbtran) [inline content/en/docs/appstore/connectors/aws/amazon-bedrock.md:75]: I'd like to add a row to this table, which is a constant specific to Amazon Bedrock:

| Any | **Timeout** | Specifies how long the Call REST service activity should wait for the REST endpoint to respond |

**Review by trongbtran — COMMENTED** [review]
Added a new constant "Timeout"

File: content/en/docs/appstore/connectors/aws/amazon-bedrock.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws/amazon-bedrock.md:75]: Done.

**Pull request merged**